05-onchain.md

```diff
@@ -410,7 +410,7 @@ HTLC-success transaction.
 The payment preimage either serves to prove payment (when the offering node is
 the originator of the payment) or to redeem the corresponding incoming HTLC from
 another peer (when the offering node is forwarding the payment). After a node has
-extracted the payment, it no longer need be concerned with the fate of the
+extracted the payment, it no longer needs to be concerned with the fate of the
```

Comment:
although not standard, `need be` is colloquially fine grammatically.

Reply:
I guess it's been used as a modal verb then.